testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/ClientPolicyBasicsTest.java

```diff
@@ -812,6 +813,61 @@ public void testClientIpAddressCondition() throws ClientRegistrationException, C
         }
     }
 
+    @Test
+    public void testSecureSessionEnforceExecutor() throws ClientRegistrationException, ClientPolicyException {
+        String policyBetaName = "MyPolicy-beta";
+        createPolicy(policyBetaName, DefaultClientPolicyProviderFactory.PROVIDER_ID, null, null, null);
+        logger.info("... Created Policy : " + policyBetaName);
+
+        createCondition("ClientRolesCondition-beta", ClientRolesConditionFactory.PROVIDER_ID, null, (ComponentRepresentation provider) -> {
```

Comment:
This is out of scope of this PR, but I wonder if we plan to have the condition, which will always evaluate to true? Or something like this... I can imagine that enforcing "state" and "nonce" parameter is something, which many deployments may want by default without a need for the client to match any special condition. WDYT? Just an idea for the future, nothing needed in this PR :-)

Reply:
@mposolda Yes, I totally agree with you. This "always true" condition is needed to realize [pre-set policy](https://github.com/keycloak/keycloak-community/blob/master/design/client-policies.md#pre-set-policy) which are adopted to all clients.
I'd like to implement this condition afterwards. Thank you.

Reply:
@tnorimat Thanks for the info